packages/serverpod/lib/src/database/database_connection_legacy.dart

```diff
@@ -0,0 +1,348 @@
+import 'dart:async';
+import 'dart:convert';
+import 'dart:typed_data';
+
+import 'package:postgres_pool/postgres_pool.dart';
+import 'package:serverpod/src/database/columns.dart';
+import 'package:serverpod/src/database/database_connection.dart';
+import 'package:serverpod/src/database/database_query.dart';
+import 'package:serverpod_serialization/serverpod_serialization.dart';
+
+import '../server/session.dart';
+import 'database_pool_manager.dart';
+import 'expressions.dart';
+import 'table.dart';
+
+typedef _LogQuery = void Function(
+  Session session,
+  String query,
+  DateTime startTime, {
+  int? numRowsAffected,
+  dynamic exception,
+  StackTrace? trace,
+});
+
+/// A connection to the database. In most cases the [Database] db object in
```

Comment:
Double check that this documentation is still valid, perhaps it should say explain how we are moving to deprecate things?